File: android/app/src/main/java/com/one9x/vartalap/Application.java
package com.one9x.vartalap;

import io.flutter.app.FlutterApplication;
import android.app.NotificationManager;
import android.content.Context;

public class Application extends FlutterApplication {
  @Override
  public void onCreate() {
    super.onCreate();
    cancelAllNotifications();
  }

  private void cancelAllNotifications() {
    NotificationManager notificationManager = (NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
    notificationManager.cancelAll();
  }
}

File: android/app/src/main/kotlin/com/one9x/vartalap/Application.kt
package com.one9x.vartalap

import io.flutter.app.FlutterApplication
import android.app.NotificationManager
import android.content.Context

class Application : FlutterApplication() {
    @Override
    override fun onCreate() {
        super.onCreate()
        cancelAllNotifications()
    }

    private fun cancelAllNotifications() {
        val notificationManager: NotificationManager = getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
        notificationManager.cancelAll()
    }
}